templates/converters_table_md.jinja

```diff
@@ -1,13 +1,15 @@
-# {{ include.name }}
+{# # {{ data_type }} Converters Table #}
 
 {{ include.comment }}
 
 | Name  | Expected input | Language | Distribution | Updated |
 | :---: | :------------  | :------  | :----------  | :-----: |
+{% set accepted_data_type = data_type %}  {# Set accepted data type to include in table #}
 {% for converter in include.members %}
-{% set converter_url = converter.url %}
-{% if converter.documentation %}
-  {% set converter_url = converter.documentation %}
-{% endif %}
-| [{{ converter.name }}]( {{ converter_url }} "{{ converter.comment | replace("\n", " ") }}" ) | {% for input in converter.expected_input %} {{ input }} <br> {% endfor %} | {% for lang in converter.language %} {{ lang }} <br> {% endfor %} | {% for entry in converter.distribution %} {% if entry.name == "pypi" %} [![PyPI version](https://badge.fury.io/py/{{ entry.url | replace("https://pypi.org/project/", "") }}.svg)]({{ entry.url }}) <br> {% elif entry.name == "dockerhub" %} [![dockerhub pulls](https://img.shields.io/docker/pulls/{{ entry.url | replace("https://hub.docker.com/r/", "")}}.svg)]({{ entry.url }}) <br> {% elif entry.name == "conda-forge" %} [![conda forge]({{ entry.url }}badges/version.svg)]({{ entry.url }}) <br> {% else %} [entry.name](entry.url) <br> {% endif %} {% endfor %} | !["GitHub last commit"](https://img.shields.io/github/last-commit/{{ converter.url | replace("https://github.com/", "") }}?style=plastic) |
-{% endfor %}
+    {% if converter.data_types is defined %}
+        {% if accepted_data_type in converter.data_types %}
```

Comment:
Can you do:

```suggestion
        {% if data_type in converter.data_types %}
```

And get rid of the `{% set accepted_data_type = data_type %}` line above?

I would expect it to work, no?